src/attachment.rs

```diff
@@ -201,3 +218,25 @@ pub fn hms_string(duration: Duration) -> String {
 #[derive(Debug, thiserror::Error)]
 #[error("Index[{0}]")]
 pub struct Index<I: std::fmt::Display>(pub I);
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn kv_macro() {
+        let foo = "Foo";
+
+        // foo: "Foo"
```

Comment:
This seems redundant. How about one of these: `key`, `value`, `item`, `field` instead of stringifying the value passed in?

```suggestion
        // foo: "Foo"
```